core/templates/dev/head/components/hint_editor_directive.html

```diff
@@ -0,0 +1,53 @@
+<div style="height: 100%;">
+    <div ng-if="!hintEditorIsOpen"
```

Comment:
Oops! I changed my text editor recently and it seems to default to 4-space indentation for some reason. I've corrected it now. Thanks!